controllers/topics.controllers.js

```diff
@@ -75,3 +77,23 @@ exports.getCommentsByArticleId = (req, res, next) => {
       next(err);
     });
 };
+
+exports.deleteCommentById = (req, res, next) => {
+  const { comment_id } = req.params;
+  checkCommentExists(comment_id)
+    .then(() => {
+      return removeCommentById(comment_id);
+    })
+    .then((rows) => {
+      if (rows.length === 1) {
+        res.status(204).send({ message: "Error Check" });
+      } else {
+        res
+          .status(207)
+          .send({ message: "multiple deletes detected", deleted: rows });
```

Comment:
You don't have a test for this 207 functionality and it's not required in the ticket so you should remove it